data/human/human missions.txt

```diff
@@ -5856,8 +5856,8 @@ mission "Quicksilver Mixup 2"
 		has "event: quicksilver mixup 2 available"
 	on offer
 		conversation
-			`As you walk through <origin>'s spaceport, somebody wearing office attire and carrying a tablet power walks up to you after ending a hurried conversation with another pilot.`
-			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to <planet> and back couldn't make it this month, some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day> and then return with the mail from <planet>? We'll pay you what he'd be paid: <payment>."`
+			`As you walk through <origin>'s spaceport, somebody wearing office attire and carrying a tablet walks up to you after ending a hurried conversation with another pilot.`
+			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to <planet> and back couldn't make it this month, due to some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day> and then return with the mail from <planet>? We'll pay you what he'd be paid: <payment>."`
```

Comment:
```suggestion
			`As you walk through <origin>'s spaceport, somebody wearing office attire and carrying a tablet walks up to you after ending a hurried conversation with another pilot.`
			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to <planet> and back couldn't make it this month, due to some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day> and then return with the mail from <planet>? We'll pay you what he'd be paid: <payment>."`
```
Not necessary; the original sentence is correct. This is more a writing style choice and not a fix.

Reply:
Actually, I could get @roadrunner56 opinion here, but the original would be corrected to: "The regular pilot who carries misposted mail to <planet> and back couldn't make it this month - some mechanical issues with his ship, I'm told."

But yes, using due to also works, but I'd also add "or so" to the sentence: "...and back couldn't make it this month, due to some mechanical issues with his ship, or so I'm told."

Reply:
About the power though... it was tablet power, and I don't think that is correct. 
Should I change it to "power tablet"?

Reply:
```suggestion
			`As you walk through <origin>'s spaceport, somebody wearing office attire and carrying a power tablet walks up to you after ending a hurried conversation with another pilot.`
			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to <planet> and back couldn't make it this month, due to some mechanical issues with his ship, or so I'm told. Would you be willing to carry this month's load there by <day> and then return with the mail from <planet>? We'll pay you what he'd be paid: <payment>."`
```